RFS/src/main/java/com/rfs/common/SourceResourceProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.rfs.common;
+
+import com.rfs.models.IndexMetadata;
+import com.rfs.models.ShardMetadata;
+
+public interface SourceResourceProvider {
```

Comment:
Thanks for including this change

Reply:
You're welcome!  Not a fan of the class name though, lol...  Gets the job done though!